Widgets/Widgets/VarmintWidgetButton.cs

```diff
@@ -47,25 +45,27 @@ private EventHandledState Ignore_OnFlick(VarmintWidget arg1, Vector2 arg2, Vecto
         //--------------------------------------------------------------------------------------
         private void Render(GameTime gameTime, VarmintWidget widget)
         {
-            var textToDisplay = (Content == null) ? "" : Content.ToString();
+            if (InhibitRendering) { return; }
 
-            if (BackgroundImage != null)
+            VarmintWidgetImage image = Content as VarmintWidgetImage;
```

Comment:
I want to handle this different.  The code you have here is adding a constraint to the button that the only thing allowed in the content is a VarmintWidgetImage.  We want to allow ANY kind of widget.   There is a clever way we can do this, and it is a little tricky.  I'll try to get it checked in before you arrive tonight.  Then you can get rid of this code.